py3status/modules/imap.py

```diff
@@ -339,27 +338,30 @@ def _get_mail_count(self):
                     if '*' in directory or '%' in directory:
                         import csv
                         ok, wildcard_dirs = self.connection.list(directory)
-                        if not ok:
+                        if ok != "OK":
                             raise imaplib.IMAP4.error("mailbox contains bad wildcard")
-                        directories.push([
+                        directories.extend([
                             next(csv.reader([dir.decode()], delimiter=' ', quotechar='"'))[-1]
                             for dir in wildcard_dirs
                         ])
                         directories.remove(directory)
-                        self.use_idle = False # idle AND wildcards are not supported
 
                 for directory in directories:
-                    self.connection.select(directory)
-                    unseen_response = self.connection.search(None, self.criterion)
-                    mails = unseen_response[1][0].split()
-                    tmp_mail_count += len(mails)
+                    ok, _ = self.connection.select('"{}"'.format(directory))
+                    if ok != "OK":
+                        if self.debug:
+                            self.py3.log("cannot select {}".format(directory))
```

Comment:
i don't know if we could scream louder at this point, like marking the module with a degraded color